src/test/java/com/metamx/common/parsers/JavaScriptParserTest.java

```diff
@@ -0,0 +1,52 @@
+/*
+* Licensed to Metamarkets Group Inc. (Metamarkets) under one
+* or more contributor license agreements. See the NOTICE file
+* distributed with this work for additional information
+* regarding copyright ownership. Metamarkets licenses this file
+* to you under the Apache License, Version 2.0 (the
+* "License"); you may not use this file except in compliance
+* with the License. You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing,
+* software distributed under the License is distributed on an
+* "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+* KIND, either express or implied. See the License for the
+* specific language governing permissions and limitations
+* under the License.
+*/
+
+package com.metamx.common.parsers;
+
+import com.google.common.collect.ImmutableMap;
+import junit.framework.Assert;
+import org.junit.Test;
+
+import java.util.Map;
+
+/**
+ */
+public class JavaScriptParserTest
+{
+  @Test
+  public void testParse()
```

Comment:
add a test for returning an array, like for a multi-valued dim? or just add `both: parts` to this existing test, as `parts` is an array.